frontend/src/pages/main/payment/components/confirm/amount/AmountField.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {AmountFieldProps} from "./props";
+import {TextField} from "@mui/material";
+import {InputHelperMessage} from "../../../../../../common/constansts/inputHelperMessage";
+import {DISABLE_FIELD} from "../../../../../../common/constansts/fields";
+
+const AmountField: React.FC<AmountFieldProps> = ({amount}) => {
```

Comment:
it already exists component `Amount` (i didn't use it, because it has logic that conflict with Confirm page), so i decided to put `AmountField` in Confirm components

Reply:
in that case name it differently